bot.py

```diff
@@ -709,11 +709,216 @@ def cmd_shoutout(match: Match[str]) -> Response:
     )
 
 
+class YouTubeVideo(NamedTuple):
+    video_id: str
+    title: str
+    playlist_id: str
+    playlist_position: int
+
+    def __repr__(self) -> str:
```

Comment:
`__repr__` is supposed to represent a programmer-readable way to build the object, this should be `__str__` at worst, but probably makes the most sense to just make this a method